spec/libraries/aws_engine_spec.rb

```diff
@@ -93,57 +93,57 @@ def contents
 
   describe 'resource_endpoint' do
     it 'return URL that includes image bucket name and region in Tokyo region' do
-      expect(AWSEngine.resource_endpoint).to eq('https://my-image-bucket.s3-ap-northeast-1.amazonaws.com')
+      expect(SlideHub::Cloud::Engine::AWS.resource_endpoint).to eq('https://my-image-bucket.s3-ap-northeast-1.amazonaws.com')
     end
     it 'return URL that includes image bucket name in us-east-1' do
-      AWSEngine.configure do |config|
+      SlideHub::Cloud::Engine::AWS.configure do |config|
         config.region = 'us-east-1'
       end
-      expect(AWSEngine.resource_endpoint).to eq('https://my-image-bucket.s3.amazonaws.com')
+      expect(SlideHub::Cloud::Engine::AWS.resource_endpoint).to eq('https://my-image-bucket.s3.amazonaws.com')
     end
     it 'return CDN URL' do
-      AWSEngine.configure do |config|
+      SlideHub::Cloud::Engine::AWS.configure do |config|
         config.cdn_base_url = 'https://sushi.example.com'
       end
-      expect(AWSEngine.resource_endpoint).to eq('https://sushi.example.com')
+      expect(SlideHub::Cloud::Engine::AWS.resource_endpoint).to eq('https://sushi.example.com')
     end
     it 'return S3 bucket name when enabling static hosting (In this case, SSL can not be used.)' do
-      AWSEngine.configure do |config|
+      SlideHub::Cloud::Engine::AWS.configure do |config|
         config.image_bucket_name = 'toro.example.com'
         config.use_s3_static_hosting = '1'
       end
-      expect(AWSEngine.resource_endpoint).to eq('http://toro.example.com')
+      expect(SlideHub::Cloud::Engine::AWS.resource_endpoint).to eq('http://toro.example.com')
     end
   end
 
   describe 'upload_endpoint' do
     it 'return URL that includes bucket name and region in Tokyo region' do
-      expect(AWSEngine.upload_endpoint).to eq('https://my-bucket.s3-ap-northeast-1.amazonaws.com')
+      expect(SlideHub::Cloud::Engine::AWS.upload_endpoint).to eq('https://my-bucket.s3-ap-northeast-1.amazonaws.com')
     end
     it 'return URL that includes bucket name in us-east-1' do
-      AWSEngine.configure do |config|
+      SlideHub::Cloud::Engine::AWS.configure do |config|
         config.region = 'us-east-1'
       end
-      expect(AWSEngine.upload_endpoint).to eq('https://my-bucket.s3.amazonaws.com')
+      expect(SlideHub::Cloud::Engine::AWS.upload_endpoint).to eq('https://my-bucket.s3.amazonaws.com')
     end
   end
 
   describe 's3_host_name' do
     it 'return hostname that includes region in Tokyo region' do
-      expect(AWSEngine.s3_host_name).to eq('s3-ap-northeast-1.amazonaws.com')
+      expect(SlideHub::Cloud::Engine::AWS.s3_host_name).to eq('s3-ap-northeast-1.amazonaws.com')
     end
     it 'return hostname that does not include region in us-east-1' do
-      AWSEngine.configure do |config|
+      SlideHub::Cloud::Engine::AWS.configure do |config|
         config.region = 'us-east-1'
       end
-      expect(AWSEngine.s3_host_name).to eq('s3.amazonaws.com')
```

Comment:
__[rubocop]__
* Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)